src/components/FieldOverviewCard/FieldOverviewCard.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { text } from '@storybook/addon-knobs';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { FieldOverviewCard as FieldOverviewCardComponent } from '.';
+import { CordIcon } from '../Icons';
+
+export default { title: 'Components/FieldOverviewCard' };
```

Comment:
```suggestion
export default { title: 'Components' };
```

Only a single story, nesting isn't necessary.